src/dataset-builder/DatasetBuilderUtils.test.ts

```diff
@@ -187,96 +186,6 @@ describe('test conversion of DatasetAccessRequest', () => {
   });
 });
 
-describe('test gettingHierarchyMap', () => {
```

Comment:
I see why we need to delete these tests but do we need to add any new ones to cover the new functionality?

Reply:
I think you're right, there should be one more test that mocks the call to `getConceptHierarchy`, let me look at adding that.